webpack.config.js

```diff
@@ -1,26 +1,54 @@
 const bundleName = 'ui-bundle'
 
-const { basename } = require('path')
+const path = require('path')
 const ZipPlugin = require('zip-webpack-plugin');
 const FaviconsPlugin = require('favicons-webpack-plugin')
 const CopyPlugin = require('copy-webpack-plugin')
 const HtmlPlugin = require('html-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+
+const htmlHeadTags = (fileName, filterKey, filterValue) => {
+  const compareTags = (htmlWebpackPlugin) => htmlWebpackPlugin.tags.headTags
+      .filter((tag) => filterKey.split('.').reduce((obj, path) => obj ? obj[path] : false, tag) === filterValue)
```

Comment:
Сложно) предлагаю затащить lodash/get
```
const get = require('lodash/get')
```

```suggestion
      .filter(tagNode => get(tagNode, filterKey) === filterValue)
```